src/alertsmanagement/azext_alertsmanagement/_help.py

```diff
@@ -1,72 +1,191 @@
-# coding=utf-8
-# --------------------------------------------------------------------------------------------
+# --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-# --------------------------------------------------------------------------------------------
-
-# pylint: disable=too-many-lines
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
 # pylint: disable=line-too-long
-from knack.help_files import helps  # pylint: disable=unused-import
+# pylint: disable=too-many-lines
 
+from knack.help_files import helps
 
-helps['monitor action-rule'] = """
+
+helps['monitor alert-processing-rule'] = """
     type: group
-    short-summary: Commands to manage action rule.
+    short-summary: Manage alert processing rule with alertsmanagement
 """
 
-helps['monitor action-rule create'] = """
+helps['monitor alert-processing-rule delete'] = """
     type: command
-    short-summary: Create an action rule
+    short-summary: "Delete an alert processing rule."
```

Comment:
May I ask why do you use double quotes in short-summary ?

Reply:
I removed all double quotes from short-summary of commands.
Should I do so for short-summary of parameters?